scanomatic/data_processing/phenotyper.py

```diff
@@ -269,14 +269,20 @@ def LoadFromState(cls, directory_path: str) -> "Phenotyper":
         """
         _p = paths.Paths()
 
-        raw_growth_data = unpickle_with_unpickler(np.load, os.path.join(
-            directory_path,
-            _p.phenotypes_input_data
-        ))
+        raw_growth_data = np.load(
+            safe_load(os.path.join(
```

Comment:
what does safe_load to / is it needed?

Reply:
It does:
```python
def safe_load(path, return_string=False):

    version_compatibility = _RefactoringPhases()
    fh = SafeProxyFileObject(path, version_compatibility)
    version_compatibility.io = fh
    if return_string:
        return os.linesep.join(fh.readlines())
    else:
        return fh
 ```
 Whether it is needed or not I can scarcely guess at, but it seems to be doing something for compatibility..?

Reply:
might might be needed, so lets leave it for now though